.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: go
 
+go_import_path: github.com/census-instrumentation/opencensus-service
+
 go:
   - 1.10.x
-
-before_script:
-  - GO_FILES=$(find . -iname '*.go' | grep -v /vendor/)  # All the .go files, excluding vendor/ if any
-
-script:
-  - if [ -n "$(gofmt -s -l $GO_FILES)" ]; then echo "gofmt the following files:"; gofmt -s -l $GO_FILES; exit 1; fi
-  - go test -v -race ./...            # Run all the tests with the race detector enabled
-  - 'if [[ $TRAVIS_GO_VERSION = 1.8* ]]; then ! golint ./... | grep -vE "(_mock|_string|\.pb)\.go:"; fi'
+ 
+install:
+  - make install-tools
```

Comment:
Could we explicitly trigger the test and fmt directives of the Makefile? `install-tools` only seems tailored to download `golint` which am not yet sold on disabling the build.

Reply:
In the presence of a Makefile Travis runs `make` so it goes to its default target. Right now it is "fmt test" so it runs gofmt and runs the tests. This step here was part of the "setup" and just needed if we wanted to run golint on Travis.